include/nbl/system/IFileArchive.h

```diff
@@ -41,6 +41,12 @@ class IArchiveLoader : public core::IReferenceCounted
 	//! Returns an array of string literals terminated by nullptr
 	virtual const char** getAssociatedFileExtensions() const = 0;
 
+	virtual bool isMountable() const = 0;
```

Comment:
they all must be mountable, so its pointless to have this method

Reply:
also `IFileArchive::readFile` is redundant I think.

Reply:
> also IFileArchive::readFile is redundant I think.

Why so? I mean it can be useful for retrieving files from archives

Reply:
because its the system that's supposed to handle all file reading etc.

Reply:
fixed?